### PR TITLE
Feature/prlc100 184/domestic exception

### PR DESCRIPTION
**Pull request by monalisasarkar1**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/PRLC100-184

### Change description ###
C100 Rebuild MIAM exceptions - domestic violence and abuse

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/main/steps/c100-rebuild/miam/domestic-abuse/content.ts [hunk 1]
```diff
@@ -0,0 +1,428 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { atLeastOneFieldIsChecked } from '../../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  section: 'Miam section',
+  caption: 'MIAM exemptions',
+  headingTitle: 'Do you have any of the following evidence of domestic violence or abuse?',
```
Comment (yogendraU) [inline src/main/steps/c100-rebuild/miam/domestic-abuse/content.ts:9]: Please add serviceName as 'Child arrangements',

**Review by yogendraU — APPROVED** [review]

File: src/main/steps/c100-rebuild/miam/domestic-abuse/content.ts (hunk 1 above)
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/domestic-abuse/content.ts:9]: Please rename 'headingTitle' to 'title'.

File: src/main/app/case/case.ts [hunk 2]
```diff
@@ -65,6 +65,7 @@ export interface Case {
   accessCode?: string;
   detailsKnown?: string;
   miam_haveDocSigned?: string;
+  miam_domesticabuse_involvement?: string;
```
Comment (channa-patil) [inline src/main/app/case/case.ts:68]: Can we rename this to 'miam_domesticAbuse' ?

File: src/main/steps/c100-rebuild/miam/domestic-abuse/content.ts [hunk 3]
```diff
@@ -0,0 +1,428 @@
...
+const languages = {
+  en,
+  cy,
+};
+
+export const form: FormContent = {
+  fields: {
+    miam_domesticabuse_involvement: {
+      id: 'miam_domesticabuse_involvement',
+      section: l => l.section,
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/domestic-abuse/content.ts:212]: Is 'section' field this required here ?

File: src/main/steps/c100-rebuild/miam/domestic-abuse/content.ts [hunk 4]
```diff
@@ -0,0 +1,428 @@
+import { TranslationFn } from '../../../../app/controller/GetController';
+import { FormContent } from '../../../../app/form/Form';
+import { atLeastOneFieldIsChecked } from '../../../../app/form/validation';
+
+// eslint-disable-next-line @typescript-eslint/explicit-module-boundary-types
+const en = () => ({
+  section: 'Miam section',
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/domestic-abuse/content.ts:7]: I am not sure if this is being used to render. Can we please remove if not being used ?

File: src/main/steps/c100-rebuild/miam/domestic-abuse/template.njk [hunk 5]
```diff
@@ -0,0 +1,11 @@
+{% extends "common/template.njk" %}
+
+
+{% block page_content %}
+<span class="govuk-caption-xl">{{caption}}</span>
+<h1 class="govuk-heading-xl govuk-!-margin-bottom-7">{{headingTitle}}</h1>
+<div class="govuk-inset-text"> {{inset1}}<br/><br/>{{inset2}} </div>
+<p class="govuk-body-m">{{detailsPara}}</p>
+<p class="govuk-body-m">{{paragraph1}}</p>
```
Comment (channa-patil) [inline src/main/steps/c100-rebuild/miam/domestic-abuse/template.njk:9]: Can we please remove these unused lines for paragraph1 & paragraph2 ?

File: src/main/steps/common/form/fields/inset.njk [hunk 6]
```diff
@@ -0,0 +1,5 @@
+{% from "govuk/components/inset-text/macro.njk" import govukInsetText %}
```
Comment (channa-patil) [inline src/main/steps/common/form/fields/inset.njk:1]: Where this common defined insert.njk is being used ?
Can we use if feasible else let's remove this please.

**Pull request merged**